src/xrEngine/line_edit_control.cpp

```diff
@@ -364,7 +365,7 @@ void line_edit_control::assign_callback(int const dik, key_state state, Callback
     m_actions[dik]->on_assign(prev_action);
 }
 
-void line_edit_control::insert_character(char c) { m_inserted[0] = c; }
+void line_edit_control::insert_character(char c) { m_inserted[0] = c; } // FIXME: increment index or use vector-like container instead
```

Comment:
In case when TI returns more that one character this will be broken

Reply:
The character will be broken? (wrong character) Or the engine will be broken? 😃